code/style.css

```diff
@@ -1,7 +1,7 @@
 /* Global css variables used for colors */
```

Comment:
I like that you have added you own styling to the project making it more personal!